readme.md

```diff
@@ -0,0 +1,69 @@
+# Hackathon Project: SenaiLabIntegration
+
+## Overview
+
+Welcome to the GitHub repository for our project developed during the 20-hour InovaTI HACKATHON. This project aims to integrate a laboratory testing machine into the lab's digital system.
+
+## The Challenge
+
+During the Hackathon, participants were tasked with creating innovative solutions for real problems local companies have. Our team focused on solving the data integrity problem of a lab test result. Senai, a local lab, tests construction materials such as concrete to verify their quality. After testing the material, a lab technician has to read the result value on the machine's display and record the result in their system, which is error-prone.
+
+## Our Solution
```

Comment:
Could we tell about this approach as a goal for a demonstration? Then make it clear that our main solution to this challage works with another strategy (headless)

Reply:
Add a note at the end of the readme.